File: cpp/home_work_1/task8.cpp
#include <iostream>

int main() {
    int x;
    std::cout << "Please enter x ";
    std::cin >> x;

    do {
        int r = x % 10;
        std::cout << "'" << r << "' ";
        x = (x - r) / 10;
    } while (x != 0);

    return 0;
}

File: java/home_work_1/task8.java
import java.util.*;
class Task1 {
    public static void main(String[] args) {
        
            int x;
            System.out.println("Please Enter number");
            Scanner sx = new Scanner(System.in);
	    x = sx.nextInt();
	    do {
		int r = x % 10;
		System.out.print( "'" + r + "' ");
		x = (x - r) / 10;
	    } while (x != 0);

    }
}
